src/applications/simple-forms/21-0845/pages/infoScope.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import {
+  AUTHORIZER_TYPES,
+  THIRD_PARTY_TYPES,
+  INFORMATION_SCOPES,
+} from '../definitions/constants';
+import { getFullNameString } from '../utils';
+
+/** @type {PageSchema} */
+export default {
+  uiSchema: {
+    informationScope: {
+      'ui:widget': 'radio',
+      'ui:options': {
+        labels: {
+          [INFORMATION_SCOPES.LIMITED]: 'Limited information',
+          [INFORMATION_SCOPES.ANY]: 'Any information',
+        },
+        updateSchema: formData => {
+          const {
+            authorizerType,
+            thirdPartyType,
+            personFullName,
+            organizationName,
+          } = formData;
+          const titleString =
+            authorizerType === AUTHORIZER_TYPES.VETERAN
+              ? 'How much information from your VA record do you authorize us to release to [third-party-name]?'
+              : 'I authorize VA to provide [third-party-name] the following information from my VA record:';
+          const thirdPartyName =
+            thirdPartyType === THIRD_PARTY_TYPES.PERSON
+              ? getFullNameString(personFullName)
+              : organizationName;
+
+          return {
+            title: (
+              <span className="vads-u-font-family--serif vads-u-font-size--h4 vads-u-font-weight--bold">
```

Comment:
any time we're working with a title, I think we need an h tag rather than a span, and then just style it how we want.

Reply:
For the same reason as above for authType page, this also needs to be a `<span/>` to avoid screwing up the Review-page's A11y content-hierarchy.